app/javascript/flavours/glitch/features/notifications/index.js

```diff
@@ -224,8 +226,37 @@ class Notifications extends React.PureComponent {
     this.props.onMarkAsRead();
   };
 
+  groupUpNotifications(notifications, types) {
+    const groupedNotifications = [];
+    for (const notif of notifications) {
+      if (!notif) {
+        groupedNotifications.push(notif);
+        continue;
+      }
+      const newNotif = notif.set('account', ImmutableList([notif.get('account')]));
+      if (types.includes(notif.get('type'))) {
+        const matchingNotifIdx = groupedNotifications.findIndex(
+          other => other.get('type') === notif.get('type') && other.get('status') === notif.get('status'),
```

Comment:
Now, this throws `Uncaught TypeError: can't access property "get", other is null`

Reply:
ah, yeah that'd make sense. still have no clue why you have a null entry, i couldn't get one to show up 🥲

Reply:
Ah I see now, null notifications appear to be used as loading gaps. I don't think I have enough notifications on my local to have loading gaps, and all of my notifications are loaded on our instance so the loading gap doesn't appear. Should be fixed with the most recent commit

Reply:
Ah, I was already looking into if my dev instance is broken again ^^
But good to know and yes, no TypeErrors after the last commit.